packages/wallets/src/AppContent.tsx

```diff
@@ -5,7 +5,6 @@ import './AppContent.scss';
 const AppContent: React.FC = () => {
     return (
         <div className='wallets-app'>
-            <div className='wallets-modal-responsive-root' id='wallets_modal_responsive_root' />
```

Comment:
@adrienne-deriv cause you deleted this div, guess you have to delete `const rootResponsiveRef = useRef<HTMLElement | null>(document.getElementById('wallets_modal_responsive_root'));` in `ModalProvider` and all related parts of code

Reply:
I'm leaving this here at the moment, because we will still need it for other modals such as Jurisdiction modal

Reply:
So to hide the Deriv.app header, now we need to:
1. Set `shouldHideDerivAppHeader` to `true` in `ModalStepWrapper`
2. Set `shouldHideDerivAppHeader` to `true` in `show(<YourModal />, { shouldHideDerivAppHeader: true }`

Reply:
Just for an update, to show the modal step wrapper with the Deriv.app header visible, just set the prop `shouldShowDerivAppHeader` in the `ModalStepWrapper` component to true